pkgs/tools/text/seehecht/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, rustPlatform, fetchFromGitea }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "seehecht";
+  version = "3.0.2";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "papojari";
+    repo = "seehecht";
+    rev = "v${version}";
+    sha256 = "sha256-x5zZEDaBmWpyvY+sKuiK4L+hc85prxCueWYUNMi9ty0=";
+  };
+
+  cargoSha256 = "sha256-mWGmMtUYeM97SM+/jtOzkAe1usuZT4yOI6PAbiGKe7M=";
+
+  postInstall = ''
+    ln -s $out/bin/seh $out/bin/seehecht
+  '';
+
+  meta = with lib; {
+    description = "A tool to quickly open a markdown document with already filled out frontmatter";
```

Comment:
```suggestion
    description = "Quickly open a markdown document with already filled out frontmatter";
```

Reply:
So, should I open another PR for this?

Reply:
If you really want to but I didn't consider this major nothing to do another review round.